adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -64,6 +52,33 @@ object AkkaHttpAdapter {
     )
   }
 
+  def makeHttpUploadService[R, E](
+    interpreter: GraphQLInterpreter[R, E],
+    skipValidation: Boolean = false,
+    enableIntrospection: Boolean = true,
+    queryExecution: QueryExecution = QueryExecution.Parallel,
+    requestInterceptor: RequestInterceptor[R] = RequestInterceptor.empty
+  )(implicit
+    runtime: Runtime[R with Random],
```

Comment:
We don't need the `Blocking` constraint anymore?

Reply:
Yeah the file reading is done by tapir internally.